quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/WidgetController.java

```diff
@@ -49,16 +54,17 @@ public WidgetController(WidgetConfigService widgetService,
     }
 
     @RequestMapping(method = POST)
-    public ResponseEntity<HalJsonResource> getWidget(@RequestBody WidgetConfig widgetConfig) {
-        WidgetConfig    createdWidgetConfig = widgetService.saveWidget(widgetConfig);
+    public ResponseEntity<HalJsonResource> create(@RequestBody WidgetConfig widgetConfig) {
+        widgetConfig.setLastModified(LocalDateTime.now());
+        WidgetConfig    createdWidgetConfig = widgetConfigRepository.save(widgetConfig);
         HalJsonResource resource            = widgetConfigMapper.toResource(createdWidgetConfig);
 
         return new ResponseEntity<>(resource, OK);
     }
 
     @RequestMapping(value = "/{widgetId}/data", method = GET)
     public ResponseEntity<List<HalJsonResource>> getData(@PathVariable String widgetId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)